{
  "PublicDateAtUSN": "0001-01-01T00:00:00Z",
  "CRD": "0001-01-01T00:00:00Z",
  "Candidate": "CVE-2019-14584",
  "PublicDate": "2019-12-31T00:00:00Z",
  "References": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-14584",
    "https://bugzilla.tianocore.org/show_bug.cgi?id=1914",
    "https://github.com/tianocore/edk2/commit/26442d11e620a9e81c019a24a4ff38441c64ba10"
  ],
  "Description": "[Unknown description]",
  "UbuntuDescription": "",
  "Notes": null,
  "Bugs": null,
  "Priority": "low",
  "DiscoveredBy": "",
  "AssignedTo": "",
  "Patches": {
    "edk2": {
      "bionic": {
        "Status": "needs-triage",
        "Note": ""
      },
      "devel": {
        "Status": "needs-triage",
        "Note": ""
      },
      "focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "groovy": {
        "Status": "needs-triage",
        "Note": ""
      },
      "precise/esm": {
        "Status": "DNE",
        "Note": ""
      },
      "trusty": {
        "Status": "ignored",
        "Note": "out of standard support"
      },
      "trusty/esm": {
        "Status": "DNE",
        "Note": ""
      },
      "upstream": {
        "Status": "needs-triage",
        "Note": ""
      },
      "xenial": {
        "Status": "needs-triage",
        "Note": ""
      }
    }
  },
  "UpstreamLinks": {}
}